src/components/pages/Navbar/Navbar.css

```diff
@@ -51,7 +51,26 @@
   padding-right: 3vw;
 }
 
+/* Mentor nav styles */
 
+.mentorStatus {
+  height: 100%;
+  width: 18%;
+  margin-left: auto;
+  display: flex;
+  justify-content: right;
+  align-items: center;
+  margin-right: 1%;
+}
+
+.ant-switch-checked {
```

Comment:
interesting, to use the same thing in Navbar that was blocking your progress in the sidebar CSS.